Intel_SighPP_CMake/Intel_SighPP_CMake/src/CMakeLists.txt

```diff
@@ -14,18 +14,19 @@ set(SRC_FILES
 
 	${PROJECT_SOURCE_DIR}/Intel_SighPP_CMake/src/classification_result.hpp
     ${PROJECT_SOURCE_DIR}/Intel_SighPP_CMake/src/setup_helper.hpp
-	${PROJECT_SOURCE_DIR}/Intel_SighPP_CMake/src/interface_controller/output_stream_controller.cpp
+	${PROJECT_SOURCE_DIR}/Intel_SighPP_CMake/src/interface_controller/output_stream_controller.cpp    
+	${PROJECT_SOURCE_DIR}/Intel_SighPP_CMake/src/interface_controller/output_stream_controller.hpp
 	${PROJECT_SOURCE_DIR}/Intel_SighPP_CMake/src/interface_controller/api_controller.cpp
 	${PROJECT_SOURCE_DIR}/Intel_SighPP_CMake/src/interface_controller/api_controller.hpp
 	${PROJECT_SOURCE_DIR}/Intel_SighPP_CMake/src/interface_controller/api_impl_websocket.cpp
 	${PROJECT_SOURCE_DIR}/Intel_SighPP_CMake/src/interface_controller/api_user_interface.hpp
     ${PROJECT_SOURCE_DIR}/Intel_SighPP_CMake/src/ML_Lib/caffe_impl.cpp
-   ${PROJECT_SOURCE_DIR}/Intel_SighPP_CMake/src/ ML_Lib/ml_impl_depth.cpp
+    ${PROJECT_SOURCE_DIR}/Intel_SighPP_CMake/src/ ML_Lib/ml_impl_depth.cpp
```

Comment:
I think we can remove this file as it is useless?

Reply:
You already did in #26. This is a merge conflict 😄 Will be fixed when the conflict is amended.